docs/advanced_usage.rst

```diff
@@ -482,18 +483,35 @@ traces should be sent off. If an error occurs, a message will be displayed in
 the console, and changes can be made as needed.
 
 
+.. _uwsgi:
+
 uWSGI
 -----
 
-The default configuration of uWSGI applications does not include the
-``--enable-threads`` setting which must be set to ``true`` for the
-tracing library to run.  This is noted in their best practices doc_.
+The tracer and profiler support uWSGI when configured with the following:
+
+- Threads must be enabled with `enable-threads <https://uwsgi-docs.readthedocs.io/en/latest/Options.html#enable-threads>`_ or with `threads <https://uwsgi-docs.readthedocs.io/en/latest/Options.html#threads>`_ if running uWSGI in multithreaded mode.
+- If manual instrumentation and configuration is used, `lazy-apps <https://uwsgi-docs.readthedocs.io/en/latest/Options.html#lazy-apps>`_ must be used.
+
+To enable tracing with automatic instrumentation and configuration with environment variables, use `import <https://uwsgi-docs.readthedocs.io/en/latest/Options.html#import>`_ option with the setting ``ddtrace.bootstrap.customize``. For example, add the following to the uWSGI configuration file::
+
+  import=ddtrace.bootstrap.sitecustomize
```

Comment:
Is there a CLI option/example we can provide as well?

Reply:
The option and CLI arguments are the same in uwsgi. There's a link to the documentation for reference.
Not against adding more details, but it's already correct imho.

Reply:
Yeah, but any place we can spell it out a little better, always good.

We generally find solid copy/pastable examples work _really_ well for onboarding. Given not everyone uses a `uwsgi.ini`, we should either show them a copy/pastable example of that... or also give CLI option examples they can work with.


For people who know uWSGI enough, probably a bit overkill, but better safe than sorry.

Reply:
Once we move out of the realm of the of `ddtrace-run` doing all the magic, specifying each "do A if B else do C because of D" to be the wrong path. We should assume that once a user is going to use code, they having to make a series of decisions that make sense for their application. That's why I think the link to the docs is the right approach. 

The problem with including a snippet of `uwsgi.ini` here is that we'd (1) be repeating ourselves with the instructions above on options based on the usage and (2) it's not meant to be copy paste since a customer could end up overriding their settings for other non-essential options we would have included for using the tracer.